lib/spoom/sorbet/translate/sorbet_assertions_to_rbs_comments.rb

```diff
@@ -120,12 +147,16 @@ def t_annotation?(node)
           return false unless t?(node.receiver)
 
           case node.name
-          when :let, :cast
-            return node.arguments&.arguments&.size == 2
+          when :let
```

Comment:
It feels a little odd to have these options checks as part of this method. This method intuitively checks if something _is_ an annotation. The fact that the translate option is turned off doesn't make the thing not be an annotation anymore.

Would it require too much work to move this out? Or maybe we could have another method that calls this one called `should_translate?`. What do you think?

Reply:
The thing is I don't want to check twice on the name of the send.

I renamed the method `translatable_annotation?` instead.